providers/k8s/connection/manifest/connection_test.go

```diff
@@ -97,7 +97,7 @@ func TestManifestDiscovery(t *testing.T) {
 	}
 	inv, err := resources.Discover(pluginRuntime, cnquery.Features{})
 	require.NoError(t, err)
-	require.Len(t, inv.Spec.Assets, 2)
+	require.Len(t, inv.Spec.Assets, 3)
```

Comment:
According to the comments in this PR, the manifest part didn't change. Why did this increase?

Reply:
we discover the namespaces now. Before we skipped over them which was actually a bug. We turned on namespace discovery for api scans, but not for manifests